lib/sequent/core/event.rb

```diff
@@ -24,14 +22,22 @@ def initialize(args = {})
 
       def payload
         result = {}
-        instance_variables.reject { |k| payload_variables.include?(k.to_s) }.each do |k|
+        instance_variables
+          .reject { |k| payload_variables.include?(k.to_s) }
```

Comment:
Maybe `payload_variables` should be an array of symbols (`%i{...}`) instead of strings? Saves a `to_s` call here.